### ISSUE TITLE
Gulp compilation task always warns about "Unknown 'mobil-menu' in $breakpoints

### ISSUE THREAD

**Issue by sigurdsvela** [Chrisriis/riiskit#3]
Out of the box, I receive this waring when compiling

![skjermbilde 2015-09-23 kl 13 24 18](https://cloud.githubusercontent.com/assets/5571884/10043868/8c36d370-61f6-11e5-9754-3398ea7099ec.png)

**Comment by Chrisriis** [comment]
Yes. It doesn't seem to cause any problems as the breakpoint still works, but I will look into it and get rid of the warning.
